.travis.yml

```diff
@@ -4,9 +4,21 @@ dist: trusty
 sudo: false
 language: c
 
+before_script: # Get vita-gcc first (for `make tests` purpose)
+  - URL=$(curl -s https://api.github.com/repos/vitasdk/autobuilds/releases | grep -oP '[^"]+download/master-linux[^"]+' | head -n 1)
```

Comment:
we cannot use `api.github.com`, api have call limit.
https://github.com/vitasdk/vita-headers/blob/master/.travis.d/download_sdk.sh#L4